.gitignore

```diff
@@ -1,6 +1,6 @@
 .idea/
 node_modules/
-lib/
+lib/*
```

Comment:
Would it be better to use `/lib/` to be consistent with the other project we built? It seems that `lib/*` would also match cases where `lib` is not in the root directory.